packages/ui/src/components/tabs.tsx

```diff
@@ -115,9 +115,10 @@ const TabsTrigger = React.forwardRef<React.ElementRef<typeof TabsPrimitive.Trigg
         className={cn(tabsTriggerVariants({ variant: context.variant ?? variant, className }))}
         {...props}
       >
-        {context.variant === 'navigation' && (
+        {/* Active tab Radial background - removed till we add different style */}
+        {/* {context.variant === 'navigation' && (
```

Comment:
This change is the request from UX team to hide the radial gradient for active tabs for time being.

![image](https://github.com/user-attachments/assets/09ddd50e-18d1-47a3-a4fd-3e0e5b48db9a)

![image](https://github.com/user-attachments/assets/94886538-394c-429c-b536-9ac63e2b8a2a)